styles.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Dosis', sans-serif
+}
+
+html{
+  background: url("80s-record-player.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
```

Comment:
background size does not require vendor prefixes - http://caniuse.com/#feat=background-img-opts

but excellent css!